HoloPipelines/pipelines/components/compSeparateNumpy.py

```diff
@@ -0,0 +1,54 @@
+# this comp generate numpy lists list from unique value from the numpy list input (comp exclusive for abdomen model at the moment)
+
+import pathlib
+from components import compNumpy2obj as makeObj
+from components import compObj2glbWrapper as makeGlb
+import numpy as np
+import sys
+import logging
+
+
+def main(inputNumpy, outputPath):
+    outputPath = pathlib.Path(outputPath)
+    # load numpy
+    if isinstance(inputNumpy, np.ndarray):
+        originalNumpy = inputNumpy
+    else:
+        try:
+            # if inputNumpy is path to npy file
+            originalNumpy = np.load(str(pathlib.Path(inputNumpy)))
```

Comment:
it's kinda wierd that your input can handle both the numpy array and also a numpy file. Just stick to one and make sure others use it correctly

Reply:
component now only accepts NumPy array